fixtures/american-dj/encore-lp12z-ip.json

```diff
@@ -0,0 +1,580 @@
+        {
+          "dmxRange": [41, 60],
+          "type": "Zoom",
+          "angle": "10deg",
+          "comment": "Narrow"
+        },
+        {
+          "dmxRange": [61, 80],
+          "type": "Zoom",
+          "angle": "30deg",
+          "comment": "Medium Flood"
+        },
+        {
+          "dmxRange": [81, 100],
+          "type": "Zoom",
+          "angle": "40deg",
+          "comment": "Wide Flood"
+        },
+        {
+          "dmxRange": [101, 255],
+          "type": "Zoom",
+          "angle": "50deg",
+          "comment": "Very Wide Flood"
+        }
+      ]
+    },
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Color Temperature": {
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "2700K",
+        "colorTemperatureEnd": "7000K",
+        "comment": "Linear"
+      }
+    },
+    "Color Temperature Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [1, 54],
+          "type": "ColorTemperature",
+          "colorTemperature": "2700K"
+        },
+        {
+          "dmxRange": [55, 109],
+          "type": "ColorTemperature",
+          "colorTemperature": "3200K"
+        },
+        {
+          "dmxRange": [110, 164],
+          "type": "ColorTemperature",
+          "colorTemperature": "4000K"
+        },
+        {
+          "dmxRange": [165, 219],
+          "type": "ColorTemperature",
+          "colorTemperature": "5600K"
+        },
+        {
+          "dmxRange": [220, 255],
+          "type": "ColorTemperature",
+          "colorTemperature": "6500K"
+        }
+      ]
+    },
+    "Dim mode": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 20],
+          "type": "Maintenance",
+          "comment": "Default to unit setting"
+        },
+        {
+          "dmxRange": [21, 40],
+          "type": "Maintenance",
+          "comment": "Standard"
+        },
+        {
+          "dmxRange": [41, 60],
+          "type": "Maintenance",
+          "comment": "Stage"
+        },
+        {
+          "dmxRange": [61, 80],
+          "type": "Maintenance",
+          "comment": "TV"
+        },
+        {
+          "dmxRange": [81, 100],
+          "type": "Maintenance",
+          "comment": "Architectural"
+        },
+        {
+          "dmxRange": [101, 120],
+          "type": "Maintenance",
+          "comment": "Theatre"
+        },
+        {
+          "dmxRange": [121, 140],
+          "type": "Maintenance",
+          "comment": "Stage 2"
+        },
+        {
+          "dmxRange": [141, 160],
+          "type": "Maintenance",
+          "parameterStart": "fast",
+          "parameterEnd": "slow",
+          "comment": "Dim speed fast to slow (0.1s - 10s)"
```

Comment:
```suggestion
          "dmxRange": [141, 160],
          "type": "Speed",
          "speedStart": "0.1s",
          "speedEnd": "1s",
          "comment": "Dimmer speed"
```

Reply:
speedStart and speedEnd doesnt allow seconds as a unit, should I convert it to Hertz?

Reply:
Oh indeed. Hertz is not really correct here, as it's not something that occurs every X seconds, but rather something that *takes* X seconds. So maybe `Time` rather than `Speed` as a capability type would be more correct?

See also https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md.

Reply:
Understood, yes, in that case Time sounds like the best option